src/package.json

```diff
@@ -37,7 +37,6 @@
   "homepage": "https://github.com/edrlab/readium-desktop",
   "dependencies": {
     "bindings": "^1.5.0",
-    "mathjax": "^3.2.2",
```

Comment:
same, I think it's better to create a dedicated commit in develop branch for this fixes

Reply:
thank you for the external PR